### DISCUSSION TITLE
Is Granite-7 the absolute lowest revision of Granite compatible, or could Granite-5 suffice?

### DISCUSSION THREAD

**Discussion by anoduck** [alainm23/planify#1399] [Q&A] [answered]
I was hoping to be able to build this app in OpenBSD, since such graphical niceties are far and few between, unfortunately I am stuck satisfying the dependency of granite-7. The current revision of granite available in the package repository is granite-5, and I am wondering if the earlier revision would satisfy the requirements needed by the application? 

Is granite required to provide a nonessential feature, or would not being able to meet the specific revision requirement break the application?

Is there an earlier release compatible with granite-5?

**Comment by alainm23** [comment, marked as answer]
Hi, I plan to remove the granite dependency, really the only thing it is used for is date formats.

**Reply by anoduck** [reply]
That's good news for me. Look forward to the next release then.
